packages/vscode-extension/src/dependency/prebuild.ts

```diff
@@ -100,7 +98,7 @@ export class Prebuild implements Disposable {
 
     const prebuildCancelToken = new CancelToken();
     const prebuildProcess = new PrebuildProcess(
-      this.runPrebuild(buildConfig, outputChannel, prebuildCancelToken),
+      this.runPrebuild(buildConfig, buildOptions),
```

Comment:
You need to pass `prebuildCancelToken` here, not the one from `buildOptions`


```suggestion
      this.runPrebuild(buildConfig, {...buildOptions, cancelToken: prebuildCancelToken}),
```